### PR TITLE
Angular 11 - JWT Authentication Example & Tutorial With PHP

### PR DESCRIPTION
**Pull request by jumamiller**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you placed your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
This fixes #1691

### CODE REVIEW

**Comment by geoffrey45** [comment]
Hello @jumamiller. The title of this article conflicts with that of another already published article. See https://www.section.io/engineering-education/getting-started-with-jwt-using-angular8-and-nodejs/. The article talks about jwt in angular 11 using a node.js server. 

Looking at your article, seems like there's some PHP used. Just a suggestion, maybe you could update the title to include PHP, to differentiate it from the other article.

File: articles/jwt-auth-angular/index.md [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+```
+Well, let's start our angular application by running the following command:  
+
+```bash
+ng serve --open //starts on port 4200 by default unless you specfied otherwise
+```
+You can now make requests to our PHP endpoint and login while the generated token is stored in your browser's local storage.  
+
+### Conclusion
+You learned how to use JWT authentication in your Angular 11 framework in this tutorial.
```
Comment (paulodhiambo) [inline articles/jwt-auth-angular/index.md:543]: Consider changing "You learned how to use JWT authentication in your Angular 11 framework in this tutorial." to "You have learned how to use JWT authentication in your Angular 11 application in this tutorial. Implement other authentication strategies such as token authentication in your Angular app."

**Review by paulodhiambo — CHANGES_REQUESTED** [review]
@jumamiller  an amazing article. Check on the suggested changes and ping me when done.

**Comment by jumamiller** [comment]
@paulodhiambo, done!

**Comment by paulodhiambo** [comment]
@jumamiller an amazing article on Angular 11 authentication. @hectorkambow  this article is ready for final review.

**Comment by hectorkambow** [comment]
@jumamiller  👋 Good afternoon - Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.
Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
Be sure to see our resources page to see more info on [plagiarism](https://www.scribbr.com/category/plagiarism/) and what is considered as such.

🤔 

[Angular 11 - JWT Authentication Example & Tutorial With PHP #1699.pdf](https://github.com/section-engineering-education/engineering-education/files/6159747/Angular.11.-.JWT.Authentication.Example.Tutorial.With.PHP.1699.pdf)

**Comment by jumamiller** [comment]
@hectorkambow, I've updated most of the boilerplate codes, you can recheck! 
cc @paulodhiambo

**Pull request merged**